common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3336,6 +3336,42 @@
     ],
     "sqlState" : "428EK"
   },
+  "TWS_ENCODER_UNSUPPORTED_PREFIX_KEY" : {
+    "message" : [
+      "Encoder doesn't support prefix key for state store provider: <stateStoreEncoder>. Please use another state store provider that supports prefix keys, or omit the prefix key."
+    ],
+    "sqlState" : "XXKST"
+  },
+  "TWS_IMPLICIT_KEY_NOT_FOUND" : {
+    "message" : [
+      "Implicit key not found for operation on stateName=<stateName>"
+    ],
+    "sqlState" : "55019"
+  },
+  "TWS_STORE_MULTIPLE_COLUMN_FAMILIES" : {
```

Comment:
Lets keep the layering intact while adding these error classes since its possible they might be shared by other streaming operators in the future ?

So for example, we can have specific error categories for different layers:
- STATE_STORE_ (for state store specific errors)
- STATE_TYPES_ (for composite types potentially)
- TWS_ (for operator specific errors etc)